README.md

```diff
@@ -114,6 +114,14 @@ When `package.json` and/or `yarn.lock` changes, such as when pulling down change
 yarn install
 ```
 
+Note, in v6, all JS packages are peer dependencies. Thus, the installer will add the packages:
```

Comment:
Maybe we should add a note on why packages are now peer dependencies to prevent questions and issues. What do you think?